### PR TITLE
Remove condition for triggering new workflow on active PR

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves: #15400

### CODE REVIEW

File: .github/workflows/regression-report.yml [hunk 1]
```diff
@@ -19,10 +19,7 @@ env:
 
 on:
   issue_comment:
-    types: [ created ]
-  pull_request:
-    types:
-      - opened
+    types: [ created, edited ]
```
Comment (romani) [inline .github/workflows/regression-report.yml:23]: We don't need "edited"

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/regression-report.yml (hunk 1 above)
Comment (relentless-pursuit) [reply .github/workflows/regression-report.yml:23]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**